third_party/docfx/templates/devsite/partials/class.header.tmpl.partial

```diff
@@ -4,7 +4,7 @@
 <div class="markdown level0 conceptual">{{{conceptual}}}</div>
 {{#inClass}}
 <div class="inheritance">
-  <h5>{{__global.inheritance}}</h5>
+  <h2>{{__global.inheritance}}</h2>

```

Comment:
I don't have a good sense of what should be a header and what should be a paragraph. This seems alright, but Assembly as a header seems out of place.

Reply:
After chatting offline, keeping them as `h2` would makes sense for *strict* accessibility guideline, however it's somewhat loosely applied as we're not using headers for aesthetic purposes but for organization and whether to keep them for TOC or not.

As they are not quite member labels and since they were `h5`/`h6` previously, it is suggested to use `h3` as per the UX suggestion. It would slightly go against the accessibility guidelines.

`<p>` is not a good replacement for headers as it "denotes a paragraph, and headers should be marked up as an actual header".

Whether we keep these as `h2` or `h3`, we have the option to prune them out of the right TOC by adding `class="hide-from-toc"` to the headers.